packages/platforms/delta/x86-64/x86-64-delta-ag9032v1/onlp/builds/src/module/src/platform_lib.c

```diff
@@ -0,0 +1,250 @@
+/************************************************************
```

Comment:
A version of this code seems to be shared amongst multiple delta platforms. 

Can we refactor it into a separate module that these platforms can all share? It is easy to setup.